template_qubuntu/02_install_groups_post.sh

```diff
@@ -4,7 +4,7 @@
 source "${SCRIPTSDIR}/vars.sh"
 source "${SCRIPTSDIR}/distribution.sh"
 
-if  [ $DIST = "trusty" ]; then
+if  [ $DIST != "trusty" ]; then
```

Comment:
Yes, and yet it builds with it, and I thought that it wouldn't build unless the pulseaudio from Launchpad was used. I'm going to close this and look again.